### PR TITLE
CMM 3 face commanders

### PR DESCRIPTION
**Pull request by Simisays**
@paulsnoops Do you have a better name for the token. Can't really find related sort of tokens

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/anikthea_hand_of_erebos.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ManaCost:2 W B G
+Types:Legendary Enchantment Creature Demigod
+PT:4/4
+K:Menace
+S:Mode$ Continuous | Affected$ Creature.Other+Enchantment+YouCtrl | AddKeyword$ Menace | Description$ Other enchantment creatures you control have menace.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever NICKNAME enters the battlefield or attacks, exile up to one target non-Aura enchantment card from your graveyard. Create a token that's a copy of that card, except it's a 3/3 black Zombie creature in addition to its other types.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigExile | Secondary$ True | TriggerDescription$ Whenever NICKNAME enters the battlefield or attacks, exile up to one target non-Aura enchantment card from your graveyard. Create a token that's a copy of that card, except it's a 3/3 black Zombie creature in addition to its other types.
+SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Enchantment.nonAura+YouCtrl | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Select target non-Aura enchantment card from your graveyard | RememberChanged$ True | SubAbility$ DBCopy
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/anikthea_hand_of_erebos.txt:9]: "up to" not handled

File: forge-gui/res/cardsfolder/upcoming/commodore_guff.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+A:AB$ Token | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | TokenAmount$ 1 | TokenScript$ r_1_1_wizard | TokenOwner$ You | SpellDescription$ Create a 1/1 red Wizard creature token with "{T}: Add {R}. Spend this mana only to cast a planeswalker spell."
+A:AB$ Draw | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | NumCards$ X | SubAbility$ DBDamage | SpellDescription$ You draw X cards and CARDNAME deals X damage to each opponent, where X is the number of planeswalkers you control.
+SVar:DBDamage:DB$ DealDamage | Defined$ Opponent | NumDmg$ X
+SVar:X:Count$Valid Planeswalker.YouCtrl
+DeckNeeds:Type$Planeswalker
+DeckHas:Ability$Token & Type$Wizard
+Oracle:At the beginning of your end step, put a loyalty counter on another target planeswalker you control.\n+1: Create a 1/1 red Wizard creature token with "{T}: Add {R}. Spend this mana only to cast a planeswalker spell."\n−3:You draw X cards and Commodore Guff deals X damage to each opponent, where X is the number of planeswalkers you control.\nCommodore Guff can be your commander.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/commodore_guff.txt:14]: Brackets missing

**Comment by paulsnoops** [comment]
Hmm maybe something like r_1_1_wizard_mana_restrict

File: forge-gui/res/cardsfolder/upcoming/commodore_guff.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Name:Commodore Guff
+ManaCost:1 U R W
+Types:Legendary Planeswalker Guff
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/commodore_guff.txt:3]: You need to remember to add new types to TypeLists.txt

**Review by tool4ever — APPROVED** [review]

**Pull request merged**